src/minimize_jdk.sh

```diff
@@ -59,6 +59,7 @@ else
 fi
 
 UNAME=$(uname -s | tr 'A-Z' 'a-z')
+JVM_OPTIONS='--enable-native-access=ALL-UNNAMED -XX:+UnlockExperimentalVMOptions -XX:+UseCompactObjectHeaders'
```

Comment:
Please add a comment here that explains why we are doing this here and not elsewhere (e.g. the client)